### PR TITLE
Enable `stable-rt` email report

### PR DESCRIPTION
**Pull request by JenySadadia**
For starters, `stable-rt` email reports will list down build, boot, and rt-tests failures.

### CODE REVIEW

File: kcidb/templates/stable_rt.j2 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{# stable-rc macros #}
+{% set selected_origins = ['maestro', 'broonie'] %}
+{% set indent = "      " %}
```
Comment (spbnick) [inline kcidb/templates/stable_rt.j2:3]: I wonder if you could merge stable_rc.j2 and stable_rt.j2 into a single stable.j2?

Comment (spbnick) [reply kcidb/templates/stable_rt.j2:3]: Are there any other ways you could extract code that's shared between these two subscriptions?

Comment (JenySadadia) [reply kcidb/templates/stable_rt.j2:3]: Another way is to use context variables while rendering the templates here https://github.com/kernelci/kcidb/blob/main/kcidb/monitor/output.py#L196.
But creating templates and importing them looks much simpler to me than tracking down context variables.

Comment (JenySadadia) [reply kcidb/templates/stable_rt.j2]: Created `common.j2` to share context variables among `stable-rc` and `stable-rt` report templates.

**Review by spbnick — CHANGES_REQUESTED** [review]
That's a good start, thank you. However, could you please rename `common.j2` to `stable.j2`. It's only "common" to stable templates, not to everything.

**Comment by spbnick** [comment]
Ah, I see you're using it for "fluster" reports too. Could you name it `maestro.j2` perhaps, then?

**Comment by JenySadadia** [comment]
> That's a good start, thank you. However, could you please rename `common.j2` to `stable.j2`. It's only "common" to stable templates, not to everything.

I wanted to use it for codec summary report as well. That's why I named it `common.j2`.

**Comment by JenySadadia** [comment]
> Ah, I see you're using it for "fluster" reports too. Could you name it `maestro.j2` perhaps, then?

We are taking into account results from `broonie` as well.

**Comment by JenySadadia** [comment]
Here is the sample `stable-rt` report:
```
Subject: KernelCI report for stable-rt: v6.1-rt@v6.1.105-rt38

OVERVIEW

        Builds: 0 passed, 2 failed

    Boot tests: 0 passed, 1 failed

      RT tests: 0 passed, 2 failed

    CI systems: maestro

REVISION

    Commit
        name: v6.1.105-rt38
        hash: 8fe1d58eff313241a0c3187db014fbccaad0b680
    Checked out from
        https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-stable-rt.git v6.1-rt


BUILDS

    Failures
      -arm64 (x86_64_defconfig)
      Build detail: https://kcidb.kernelci.org/d/build/build?orgId=1&var-id=maestro:66969c5faa07c494f8d22ac1
      -arm64 (defconfig)
      Build detail: https://kcidb.kernelci.org/d/build/build?orgId=1&var-id=maestro:66bf585a8ac59a075e6975f2
      Build error: make: *** [arch/arm64/Makefile:170: vdso_prepare] Error 2
      CI system: maestro


BOOT TESTS

    Failures

      arm64:(x86_64_defconfig)
      -acer-cbv514-1h-34uz-brya
      CI system: maestro

RT TESTS

    Failures

      rt-tests.cyclicdeadline:
      arm64:(defconfig)
      -bcm2711-rpi-4-b
      CI system: maestro

      rt-tests.cyclicdeadline.t0-avg-latency:
      arm64:(defconfig)
      -imx6q-sabrelite
      CI system: maestro

See complete and up-to-date report at:

    https://kcidb.kernelci.org/d/revision/revision?orgId=1&var-git_commit_hash=8fe1d58eff313241a0c3187db014fbccaad0b680&var-patchset_hash=


Tested-by: kernelci.org bot <bot@kernelci.org>

Thanks,
KernelCI team
```
Cc: @helen-fornazier

**Comment by helen-fornazier** [comment]
@JenySadadia the sample is looking good to me, thanks!

**Comment by JenySadadia** [comment]
As per suggestion from @spbnick , created common templates for `stable-rc` and `stable-rt` reports.

**Comment by JenySadadia** [comment]
Here is the updated sample report with test links:
```
BUILDS

    Failures
      -arm64 (x86_64_defconfig)
      Build detail: https://kcidb.kernelci.org/d/build/build?orgId=1&var-id=maestro:66969c5faa07c494f8d22ac1
      -arm64 (defconfig)
      Build detail: https://kcidb.kernelci.org/d/build/build?orgId=1&var-id=maestro:66bf585a8ac59a075e6975f2
      Build error: make: *** [arch/arm64/Makefile:170: vdso_prepare] Error 2
      CI system: maestro


BOOT TESTS

    Failures

      arm64:(x86_64_defconfig)
      -acer-cbv514-1h-34uz-brya
      Test details:
      - https://kcidb.kernelci.org/d/test/test?orgId=1&var-id=maestro:6696c76faa07c494f8d2828c
      - https://kcidb.kernelci.org/d/test/test?orgId=1&var-id=maestro:6696c76faa07c494f8d2828f

      CI system: maestro

RT TESTS

    Failures

      rt-tests.cyclicdeadline:
      arm64:(defconfig)
      -bcm2711-rpi-4-b
      Test details:
      - https://kcidb.kernelci.org/d/test/test?orgId=1&var-id=maestro:6696c76faa07c494f8d2828d

      CI system: maestro

      rt-tests.cyclicdeadline.t0-avg-latency:
      arm64:(defconfig)
      -imx6q-sabrelite
      Test details:
      - https://kcidb.kernelci.org/d/test/test?orgId=1&var-id=maestro:6696c76faa07c494f8d2828e

      CI system: maestro

See complete and up-to-date report at:

    https://kcidb.kernelci.org/d/revision/revision?orgId=1&var-git_commit_hash=8fe1d58eff313241a0c3187db014fbccaad0b680&var-patchset_hash=


Tested-by: kernelci.org bot <bot@kernelci.org>

Thanks,
KernelCI team
```

**Comment by JenySadadia** [comment]
@spbnick Can we merge this?

**Comment by spbnick** [comment]
Sure!

**Pull request merged**